### PR TITLE
`apps.c`: fix various coding style nits found by `check-format.pl`

### PR DESCRIPTION
**Pull request by DDvO**
Part of the nits were found by the enhancements to the tool in #17434.

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

**Comment by DDvO** [comment]
Thanks @paulidale for quickly approving this 

I meanwhile found that I have to rebase this PR,
and pondering about the remaining issues reported by `check-format.pl`,
I figured it would be good to fix also the remaining issues it reported,
which was not as easy as the ones in the first commit.
In particular, the tool warns on function bodies more than 200 lines long,
and indeed `load_key_certs_crls()` was more complex/lengthy than necessary, which I fixed.

Now `check-format.pl` is entirely happy with `apps.c` :smiley:

**Comment by paulidale** [comment]
200 line functions won't be close to enough for many of the commands :(

File: apps/lib/apps.c [hunk 1]
```diff
@@ -829,7 +832,18 @@ static const char *format2string(int format)
 }
 
 /* Set type expectation, but clear it if objects of different types expected. */
-#define SET_EXPECT(expect, val) ((expect) = (expect) < 0 ? (val) : ((expect) == (val) ? (val) : 0))
+#define SET_EXPECT(val) \
+    (expect = expect < 0 ? (val) : (expect == (val) ? (val) : 0))
+#define SET_EXPECT1(pvar, val) \
+    if ((pvar) != NULL) { \
```
Comment (paulidale) [inline apps/lib/apps.c:838]: For an internal macro like this, we often dispense with the parenthesis around the arguments.  Not a significant point and not a request to change anything.

**Review by paulidale — APPROVED** [review]

**Comment by DDvO** [comment]
> 200 line functions won't be close to enough for many of the commands :(

I am very aware of this :(
Indeed, many of source files in `apps*.c` would deserve a major cleanup,
not only regarding their overly lengthy `*_main()` functions.

**Comment by DDvO** [comment]
Merged - thanks @paulidale

**Pull request closed without merging**